_topic_maps/_topic_map.yml

```diff
@@ -3354,13 +3354,15 @@ Topics:
       File: oadp-using-data-mover-for-csi-snapshots-doc
     - Name: Using OADP 1.2 Data Mover with Ceph storage
       File: oadp-12-data-mover-ceph-doc
-  - Name: OADP 1.3 Data Mover
+  - Name: OADP 1.3 Data Mover and later
```

Comment:
Looking at the "  - Name: OADP 1.2 Data Mover" entry; this did not state "and later" so is a 1.4 not planned?

Reply:
Yes you are right @dfitzmau. We will be retiring the 1.2 documentation in the next few months and renaming the assembly as 'OADP Data Mover'. Thanks.